src/fabric.c

```diff
@@ -834,6 +837,10 @@ void fi_ini(void)
 			"(default: false)");
 	fi_param_get_bool(NULL, "av_remove_cleanup", &ofi_av_remove_cleanup);
 
+	fi_param_define(NULL, "offload_coll_provider", FI_PARAM_STRING,
+			"The name of a colective offload provider (default: empty - no provider)");
```

Comment:
Please limit line lengths to about 80 characters.

Reply:
Done